### PR TITLE
CP-3071 [E2E][Help] Q4 2024 Change - Verify Help Contents, Search and Contact button

### PR DESCRIPTION
**Pull request by tomaszmaleckimudita**
JIRA Reference: [CP-3071]

### :memo: Description ️
https://appnroll.atlassian.net/browse/CP-3071

[CP-3071]: https://appnroll.atlassian.net/browse/CP-3071?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by robertmudi — DISMISSED** [review]

**Comment by robertmudi** [comment]
COMMITED CHANGE 

public get helpMainHeader() {
    return $('[data-testid="help-main-header"]');  // Fixed unclosed selector bracket

**Review by robertmudi — CHANGES_REQUESTED** [review]
apps/mudita-center-e2e/src/page-objects/help.page.ts

 public get helpSubCategoriesListItemsRightColumn() {
    return $$('[data-testid="help-subcategories-list"]>div')[0].$$('[data-testid="help-subcategories-list-item"]')
  }
  
  shouldn't index be different and changed to "1" here as it is right column and left column index is "0" ?

File: apps/mudita-center-e2e/src/page-objects/help.page.ts [hunk 1]
```diff
@@ -7,40 +7,59 @@ import { ChainablePromiseElement } from "webdriverio"
...
+    return $('[data-testid="help-subcategories-list"]')
+  }
+  public get helpSubCategoriesListItems() {
+    return $$('[data-testid="help-subcategories-list-item"]')
+  }
+  public get helpSubCategoriesListItemsLeftColumn() {
+    return $$('[data-testid="help-subcategories-list"]>div')[0].$$('[data-testid="help-subcategories-list-item"]')
+  }
+  public get helpSubCategoriesListItemsRightColumn() {
+    return $$('[data-testid="help-subcategories-list"]>div')[0].$$('[data-testid="help-subcategories-list-item"]')
```
Comment (robertmudi) [inline apps/mudita-center-e2e/src/page-objects/help.page.ts:56]: shouldn't you update the index to 1 as it is the right column and left has already index "0" ?

**Review by robertmudi — CHANGES_REQUESTED** [review]

File: apps/mudita-center-e2e/src/page-objects/help.page.ts (hunk 1 above)
Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/page-objects/help.page.ts]: https://github.com/mudita/mudita-center/pull/2052/commits/9dab1d4d7d9e2e221e4d5ca57354c32a7058bd4d

File: apps/mudita-center-e2e/src/specs/help/help-section-check.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+    //List of articles should not be empty in any of the categories
+    let helpSubCategoriesListItem
+    for await (helpSubCategoriesListItem of helpSubCategoriesListItems) {
+      await expect(helpSubCategoriesListItem.$$('[data-testid="help-subcategory-articles-list-item"]')).toBeElementsArrayOfSize({ gte: 1 })
+    }
+  })
+  it("Search for questions and verify results", async () => {
+    const helpSearchInput = await HelpPage.helpSearchInput
+    helpSearchInput.setValue("How to do factory reset on Pure")
```
Comment (robertmudi) [inline apps/mudita-center-e2e/src/specs/help/help-section-check.e2e.ts:109]: await is missing

**Review by robertmudi — CHANGES_REQUESTED** [review]

File: apps/mudita-center-e2e/src/specs/help/help-section-check.e2e.ts (hunk 2 above)
Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/specs/help/help-section-check.e2e.ts:109]: Good catch thank you!

**Review by robertmudi — APPROVED** [review]

**Pull request merged**